framework/docker/setup.go

```diff
@@ -62,26 +60,9 @@ func DockerSetup(t DockerSetupTestingT) (*client.Client, string) {
 	DockerCleanup(t, cli)()
 
 	name := fmt.Sprintf("%s-%s", consts.CelestiaDockerPrefix, random.LowerCaseLetterString(8))
-	octet := uint8(rand.Intn(256))
-	baseSubnet := fmt.Sprintf("172.%d.0.0/16", octet)
-	usedSubnets, err := getUsedSubnets(cli)
-	if err != nil {
-		panic(fmt.Errorf("failed to get used subnets: %v", err))
-	}
-	subnet, err := findAvailableSubnet(baseSubnet, usedSubnets)
-	if err != nil {
-		panic(fmt.Errorf("failed to find an available subnet: %v", err))
-	}
 	network, err := cli.NetworkCreate(context.TODO(), name, network.CreateOptions{
 		Driver: "bridge",
-		IPAM: &network.IPAM{
-			Config: []network.IPAMConfig{
-				{
-					Subnet: subnet,
-				},
-			},
-		},
-
```

Comment:
I noticed when running tests in parallel there were ip overlaps, I don't think we need to manually choose an ip range, we can just let docker do that.